.gitignore

```diff
@@ -37,3 +37,6 @@ lab/topologies/lab00_cisco_base/src/netbox_data
 
 lab/topologies/*/operational_configs/*
 !lab/topologies/*/operational_configs/.gitkeep
+
+.idea/
+vm_images/
```

Comment:
I'd suggest tracking the folder and adding inside `.gitkeep` or tiny `README.md` file, so the users are not confused when asked to
>  put into `../vm_images` Cisco IOS
since there's no folder with such name.

Reply:
fix